packages/asc-ui/src/components/Blog/BlogContent/BlogContent.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react'
 import BlogContentStyle from './BlogContentStyle'
 
-const BlogContent: React.FC<{}> = ({ children }) => (
-  <BlogContentStyle>{children}</BlogContentStyle>
-)
+const BlogContent: React.FC<React.HTMLAttributes<HTMLElement>> = ({
```

Comment:
Replace HTMLAttributes type, which is huge with a small custom type/interface containing just the className